### PR TITLE
Fullstory v2 docs

### PR DESCRIPTION
**Pull request by mattgrosvenor-fs**
### Proposed changes

Updating documentation for new Actions being added to support our V2 API.

### Merge timing

This is related to new Actions being added. This should be merged after that PR is merged. See link below.

### Related issues (optional)

https://github.com/segmentio/action-destinations/pull/1431

### CODE REVIEW

File: src/connections/destinations/catalog/actions-fullstory-cloud/index.md [hunk 1]
```diff
@@ -48,8 +50,11 @@ The FullStory cloud mode destination sends information about your users and rela
 
 ### Why am I getting a ‘404 Not Found’ error?
 
-The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
+This applies to the orginal 'Identify User' and 'Track Event' actions. The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-fullstory-cloud/index.md:53]:
```suggestion
If you are using the original 'Identify User' and 'Track Event' actions and encounter a "404 Not Found" error, the user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
```

File: src/connections/destinations/catalog/actions-fullstory-cloud/index.md [hunk 2]
```diff
@@ -32,7 +34,7 @@ The FullStory cloud mode destination sends information about your users and rela
 
 ### Getting Started
 
-1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory.
+1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory. You need an `Admin` or `Architect` API Key for GDPR deletions to propagate. You may also contact FullStory directly for deletions.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-fullstory-cloud/index.md:37]: The GDPR deletions information seems important - let's pull it out and add it to the "Troubleshooting" section for visibility!

Is there a minimum level of access you would need? For example, would you need a key with architect or admin permissions to link to the destination within Segment?

```suggestion
1. You need a FullStory API Key to use the FullStory cloud mode destination. Refer to [this article](https://help.fullstory.com/hc/en-us/articles/360052021773-Managing-API-Keys){:target="_blank"} to learn how to generate a new API Key within FullStory.
```

File: src/connections/destinations/catalog/actions-fullstory-cloud/index.md [hunk 3]
```diff
@@ -48,8 +50,11 @@ The FullStory cloud mode destination sends information about your users and rela
...
 ### Why am I getting a ‘404 Not Found’ error?
 
-The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
+This applies to the orginal 'Identify User' and 'Track Event' actions. The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
 
 Data sent server-side for users must match an already existing userId that was sent from a client-side connection.
 
+The new 'Identify User V2' and 'Track Event V2' actions will automatically create users when a user matching the
+provided UID is not found.
+
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-fullstory-cloud/index.md:59]:
```suggestion

### Why can't I propagate GDPR deletions?

GDPR deletions require an `Admin` or `API` key to propagate. You may also contact FullStory directly for deletions. 

```

**Review by forstisabella — APPROVED** [review]
A couple of small suggestions, but otherwise this looks good to go!

I'll hold off on merging until the PR in the action-destinations repo has been merged.

**Comment by forstisabella** [comment]
Looks like the dev PR got merged today - I'll make sure this goes out today too!

**Comment by joshrynneFS** [comment]
Hi @forstisabella - I see our updates to the "Overview" section went live, but I'm not seeing references to the 2 new actions in the "Available Actions" section. Do we need to do anything special to get those new actions to appear here?

Image 2023-08-10 at 2 06 29 PM

**Comment by joshrynneFS** [comment]
Hi, never mind! They are appearing today. Thanks!

**Pull request merged**